api/swagger.yaml

```diff
@@ -312,6 +367,40 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /runners/{runnerId}/files/{filename}:
```

Comment:
Would it be possible to acquire a file located at `../flag.txt`? Or is that not a valid use case for our system?

Reply:
I find both approaches valid, but would go with the one I prefer: Using the `/runners/{runnerId}/files/raw` URL and a query parameter. This does not mingle the response scheme and makes it easier to extend the use case later (think of a zip compressed file download service, located under `/runners/{runnerId}/files/zipped`, for example).